src/metric-engine/src/engine.rs

```diff
@@ -164,7 +164,16 @@ impl RegionEngine for MetricEngine {
             }
             RegionRequest::Flush(req) => self.inner.flush_region(region_id, req).await,
             RegionRequest::Delete(_) | RegionRequest::Truncate(_) => {
```

Comment:
Truncate should be more complicated. We may need to truncate the meta region and clear all memory states related to the meta region.

Reply:
OK will do this